2024-11-23 16:24:48,994 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:24:49,842 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:25:55,230 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'luckyme.catering@example.com', 'contactNumber': '0905985854564', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:25:55,564 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:25:55,566 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:26:16,465 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'lucky.me@example.com', 'contactNumber': '0905985854564', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:26:16,521 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:26:16,522 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:26:18,862 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'lucky.me@example.com', 'contactNumber': '0905985854564', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:26:18,913 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:26:18,914 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:26:19,007 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'lucky.me@example.com', 'contactNumber': '0905985854564', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:26:19,064 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:26:19,065 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:26:19,141 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'lucky.me@example.com', 'contactNumber': '0905985854564', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:26:19,187 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:26:19,188 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:26:19,277 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'lucky.me@example.com', 'contactNumber': '0905985854564', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:26:19,326 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:26:19,327 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:27:42,638 INFO: Received data: {'firstName': 'Jolly', 'lastName': 'Bog', 'email': 'jollybog@example.com', 'contactNumber': '090586445255', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8010', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:27:42,695 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:27:42,696 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:27:51,346 INFO: Received data: {'firstName': 'Jolly', 'lastName': 'Bog', 'email': 'jollybog@example.com', 'contactNumber': '090586445255', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8010', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:27:51,443 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:27:51,445 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:27:51,482 INFO: Received data: {'firstName': 'Jolly', 'lastName': 'Bog', 'email': 'jollybog@example.com', 'contactNumber': '090586445255', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8010', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:27:51,536 ERROR: Error creating user: column "first_name" of relation "users" does not exist
LINE 1: INSERT INTO users (first_name, last_name, email, contact_num...
                           ^
 [in E:\eims-admin_project\eims_admin_backend\app\models.py:52]
2024-11-23 16:27:51,537 ERROR: User creation failed due to duplicate email [in E:\eims-admin_project\eims_admin_backend\app\routes.py:158]
2024-11-23 16:29:08,890 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:29:14,657 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:29:32,662 INFO: Received data: {'firstName': 'Jolly', 'lastName': 'Bog', 'email': 'jollybog@example.com', 'contactNumber': '090586445255', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8010', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:29:32,799 INFO: User data validated and inserted into the database successfully [in E:\eims-admin_project\eims_admin_backend\app\routes.py:155]
2024-11-23 16:30:03,138 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:30:11,113 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:31:18,004 INFO: Received data: {'firstName': 'Lucky', 'lastName': 'Me', 'email': 'luckyme@catering.com', 'contactNumber': '09050864544', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:31:18,451 INFO: Vendor data inserted successfully [in E:\eims-admin_project\eims_admin_backend\app\routes.py:151]
2024-11-23 16:31:18,451 INFO: User data validated and inserted into the database successfully [in E:\eims-admin_project\eims_admin_backend\app\routes.py:155]
2024-11-23 16:34:17,056 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-23 16:35:23,463 INFO: Received data: {'firstName': 'Jolly', 'lastName': 'Bog', 'email': 'jollybog@example.com', 'contactNumber': '09058654441', 'password': '1234', 'country': 'Philippines', 'city': 'CDO', 'street': 'Carne Norte', 'user_type': 'vendor', 'service': 'catering', 'minPrice': '8000', 'maxPrice': '20000'} [in E:\eims-admin_project\eims_admin_backend\app\routes.py:109]
2024-11-23 16:35:23,909 INFO: Vendor data inserted successfully [in E:\eims-admin_project\eims_admin_backend\app\routes.py:151]
2024-11-23 16:35:23,910 INFO: User data validated and inserted into the database successfully [in E:\eims-admin_project\eims_admin_backend\app\routes.py:155]
2024-11-24 22:33:17,543 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
2024-11-24 22:33:18,132 INFO: Application startup [in E:\eims-admin_project\eims_admin_backend\run.py:33]
